src/PluginUpdate.php

```diff
@@ -99,13 +99,20 @@ public function update_plugin_update_version_option(): void {
 	 * List of update procedures
 	 *
 	 * @since 1.0.10
+	 * @since x.x.x Updates procedures organized in an array by plugin version.
 	 * @return array List of update procedures names.
 	 */
 	private function update_procedures() {
 		return array(
```

Comment:
Please add to the log in `perform_plugin_update_procedure` which procedure has failed. It should have been added there initially but sine we are fixing the flow we may as well update that also.